optaplanner-docs/src/main/asciidoc/LocalSearch/LocalSearch-chapter.adoc

```diff
@@ -476,6 +476,46 @@ Use a lower tabu size than in a pure Tabu Search configuration.
   </localSearch>
 ----
 
+[[greatDeluge]]
+== Great Deluge
+
+
+[[greatDelugeAlgorithm]]
+=== Algorithm Description
+
+Great Deluge algorithm is very similar to the Simulated Annealing algorithm, it evaluates only a few moves per steps,
```

Comment:
Great Deluge is far more similar to Late Acceptance than it is to Simulated Annealing. But yes, all those 3 are fast-stepping algorithms.

Reply:
When mentioning another algorithm, link it. So <<simulatedAnnealing,Simulated Annealing>>